components/gitpod-db/src/typeorm/webhook-event-db-impl.ts

```diff
@@ -51,4 +51,21 @@ export class WebhookEventDBImpl implements WebhookEventDB {
         query.limit(limit);
         return query.getMany();
     }
+
+    public async deleteOldEvents(ageInDays: number, limit?: number): Promise<void> {
+        const repo = await this.getRepo();
+        const d = new Date();
+        d.setDate(d.getDate() - ageInDays);
+        const expirationDate = d.toISOString();
+        const LIMIT = typeof limit === "number" ? ` LIMIT ${limit} ` : "";
+        await repo.query(
```

Comment:
Would be great to use the ORM here. One similar example:
```
        const repo = await this.getWorkspaceRepo();
        const qb = repo
            .createQueryBuilder("ws")
            .select(["ws.id", "ws.ownerId"])
            .where(`ws.contentDeletedTime != ''`)
            .andWhere(`ws.contentDeletedTime < :minPurgeTime`, { minPurgeTime })
            .limit(limit);
        return await qb.getMany();
``

Reply:
I see no point in it, but let's do it!

Reply:
done